src/styles/components/_usage-guidelines.scss

```
@@ -0,0 +1,42 @@
+$breakpoint-tablet: 1220px;
+
+.cmp-usage-guidelines {
+  background-color: #01f5ac;
+
+  &__heading {
+    position: relative;
+    font-family: Viga, sans-serif;
+    font-weight: 400;
+    font-size: map.get($font-sizes, "md");
+  }
```

Add padding to fix heading spacing when on smaller displays. (RIght now it goes flush against the left screen)
```suggestion
  padding: 6rem 0 0 3rem;
  }
```